_config.yml

```diff
@@ -50,6 +50,15 @@ sidebar_categories:
     - docs/advanced/production-ready/logging
     - docs/advanced/production-ready/checklist-before-production
     - docs/advanced/production-ready/sitemap
+  Payments:
```

Comment:
Not sure about this place in the menu. I'd have put it below magento1/magento2 sections.

Reply:
Let's give it a try!